geth/common/utils.go

```diff
@@ -157,9 +157,8 @@ func Fatalf(reason interface{}, args ...interface{}) {
 func CreateTransaction(ctx context.Context, args SendTxArgs) *QueuedTx {
 	return &QueuedTx{
 		ID:      QueuedTxID(uuid.New()),
-		Hash:    common.Hash{},
 		Context: ctx,
 		Args:    args,
-		Done:    make(chan struct{}),
+		Result:  make(chan RawCompleteTransactionResult, 1),
```

Comment:
Generally, I am not a fan of using a buffered channel to pass the result because it can be read only once. I assume it's used instead of mutex and getters/setters. On the other hand, if it works fine now and is not something that limits us then let's leave it.